src/xenia/patcher/patcher.h

```diff
@@ -0,0 +1,81 @@
+/**
+ ******************************************************************************
+ * Xenia : Xbox 360 Emulator Research Project                                 *
+ ******************************************************************************
+ * Copyright 2020 Ben Vanik. All rights reserved.                             *
+ * Released under the BSD license - see LICENSE in the root for more details. *
+ ******************************************************************************
+ */
+
+#ifndef XENIA_PATCHER_H_
+#define XENIA_PATCHER_H_
+
+#include <map>
+#include <regex>
+#include <vector>
+
+#include "third_party/cpptoml/include/cpptoml.h"
+#include "third_party/xxhash/xxhash.h"
+
+#include "xenia/memory.h"
+
+struct patchDataEntry {
```

Comment:
Apart from incorrect case of the first letter, this should be in the xe::patcher namespace too.